spec/unit/use_cases/save_customer_details_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+describe SaveCustomerDetails do
+  it 'uses the customer gateway to save customer details' do
+    s = spy
```

Comment:
s

Reply:
This comment about an unclear variable name is unclear...